likes/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from likes.views import DislikeView, LikeView
+
+urlpatterns = [
+    path("like/", LikeView.as_view({"post": "post"}), name="set-like"),
```

Comment:
An alternative solution is to have `POST` and `DELETE` requests on `/posts/{post_id}/like` endpoint

Reply:
Done on `/posts/{post_id}/likes` with query parameter `like` to know, if it's like or dislike